packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -842,6 +842,19 @@ class RenderIntrinsicHeight extends RenderProxyBox {
   }
 }
 
+/// Excludes the child from baseline computations in the parent.
+class RenderIgnoreBaseline extends RenderProxyBox {
+  /// Create a render object that causes the parent to ignore the child for baseline computations.
+  RenderIgnoreBaseline({
+    RenderBox? child,
```

Comment:
Can this be `super.child`?

Reply:
unfortunately not, it's a positional argument in the superclass

Reply:
Ah I missed that it was positional.